core/src/main/java/tc/oc/pgm/api/region/Region.java

```diff
@@ -75,4 +77,6 @@
   public Iterator<BlockVector> getBlockVectorIterator();
 
   public Iterable<BlockVector> getBlockVectors();
+
+  public Collection<Class<? extends Event>> getRelevantEvents();
```

Comment:
Can this be `Iterable`?

Reply:
`Iterables` are a pain to work with, you can literally only get an iterator from them, can't know size, can't easily stream (without going thru stream support hoops), and they're generally not any better to return than a collection, when we **know** it's a collection.

I know the intent here is to return the most abstract type not to tie the implementation to any specific type, but Collection is still a very non-specific type (eg: it's not a List or Set), and is way nicer to work with than iterable

Reply:
That's fine with me.